server/router/test.js

```diff
@@ -28,13 +28,16 @@ describe('GET /conference', () => {
   });
 });
 
-describe('GET /community', () => {
-  it('/renders OK', () => {
-    useDummyData();
-    return request(app)
-      .get('/community')
-      .expect(200);
-  });
-});
+// describe.only('GET /community', () => {
```

Comment:
And if you've changed the data that RB consumes perhaps the dummy data in the tests is not out of date?